src/render/OpenGL.cpp

```diff
@@ -59,6 +59,30 @@ CHyprOpenGLImpl::CHyprOpenGLImpl() {
     m_tGlobalTimer.reset();
 }
 
+static void CHyprOpenGLImpl::logError(const GLuint& shader, bool program = false) {
+    GLint maxLength = 0;
+    if (program) {
```

Comment:
style nit: no {} around short ifs, here and below

Reply:
Should be good now